nestkernel/node.cpp

```diff
@@ -398,6 +404,26 @@ Node::sends_secondary_event( DelayedRateConnectionEvent& )
   throw IllegalConnection( "The source node does not support delayed rate output." );
 }
 
+void
```

Comment:
Personally I would rather use ``dynamic_cast`` to check if the node type is eprop-related, so that these methods do not have to be added to the Node base class. This would take a bit of re-engineering, so I'll just leave it here as a suggestion.